Source/WebKit/WebProcess/WebCoreSupport/WebPopupMenu.cpp

```diff
@@ -110,7 +110,7 @@ void WebPopupMenu::show(const IntRect& rect, LocalFrameView* view, int selectedI
     PlatformPopupMenuData platformData;
     setUpPlatformData(pageCoordinates, platformData);
 
-    WebProcess::singleton().parentProcessConnection()->send(Messages::WebPageProxy::ShowPopupMenu(pageCoordinates, static_cast<uint64_t>(m_popupClient->menuStyle().textDirection()), items, selectedIndex, platformData), m_page->identifier());
+    WebProcess::singleton().parentProcessConnection()->send(Messages::WebPageProxy::ShowPopupMenuFromFrame(view->frame().frameID(), pageCoordinates, static_cast<uint64_t>(m_popupClient->menuStyle().textDirection()), items, selectedIndex, platformData), m_page->identifier());
```

Comment:
view should be a LocalFrameView& if we assume it's non-null, but maybe best for a separate PR.